README.md

```diff
@@ -63,32 +62,32 @@ Expected JSON response from `GET /cards/card001/gt`:
       "title": "Giant"
     }
   ],
-  "backgroundImageUrl": "/front-cover-portrait-1.jpg",
-  "price": "£4.00",
+  "imageUrl": "/front-cover-portrait-1.jpg",
+  "price": 400,
```

Comment:
👍  We need some instruction in the README around what this now represents (i.e. price in pennies (sterling))